src/script/main/app.ts

```diff
@@ -372,6 +372,11 @@ export class App {
       onProgress(2.5);
       telemetry.timeStep(AppInitTimingsStep.RECEIVED_ACCESS_TOKEN);
 
+      const selfUser = await this.repository.user.getSelf([{position: 'App.initiateSelfUser', vendor: 'webapp'}]);
+
+      await initializeDataDog(this.config, selfUser.qualifiedId);
```

Comment:
We first fetch (only fetching, not initializing at all) the self user, then we can use it to enable datadog